hook-common/src/pgqueue.rs

```diff
@@ -827,18 +844,18 @@ mod tests {
         "https://myhost/endpoint".to_owned()
     }
 
-    #[tokio::test]
-    async fn test_can_dequeue_job() {
+    #[sqlx::test(migrations = "../migrations")]
```

Comment:
This is compatible with `#[tokio::test]` but it gives us a DB per test.

We shouldn't have to provide `migrations` but it was giving me an error without it, maybe something to do with using workspaces? It didn't seem worth investigating right now.

Reply:
We can take a look later; I wasn't familiar with `sqlx::test`, good suggestion!

Reply:
It does mean we need to have the database url set in the environment, so perhaps we could document that in the README